packages/react-scripts/bin/react-scripts.js

```diff
@@ -11,6 +11,7 @@
 'use strict';
 
 var spawn = require('cross-spawn');
+var path = require('path');
```

Comment:
Is this needed? I don't see where in what you've added that you're using `path`.